bin/git-sync

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+function _usage()
+{
+  local command="git sync"
+  cat << EOS
+Usage:
+  ${command} <remote> <branch>
+  ${command} -h | --help | help | ?
+
+Sync local branch with <branch> of <remote>.
+All changes and untracked files and directories will be removed.
+
+Example:
+  ${command} origin master
+EOS
+}
+
+function main()
+{
+  while [ "$1" != "" ]; do
+    case $1 in
+      -h | --help | help | "?" )
```

Comment:
As we don't use "?" for seeing usage with any other commands, we shouldn't do it here either.
`?` is a special character in some terminals with `extglob` (i think) enabled, and could confuse people as well.